### PR TITLE
Use a separate url for replies for sms transport

### PR DESCRIPTION
**Pull request by justinvdm**
From the API docs:

```
If MO Message ID is validated, MT will not be charged.
Only one free MT is allowed for each MO.
```

I only noticed now that replies should be sent over a different URL for this.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by justinvdm** [comment]
Ready for review.

File: vxvas2nets/vas2nets_sms.py [hunk 1]
```diff
@@ -68,6 +72,14 @@ def get_message_dict(self, message_id, vals):
             'transport_metadata': {'vas2nets_sms': {'msgid': vals['msgid']}}
         }
 
+    def get_outbound_url(self, message):
+        id = get_in(message, 'transport_metadata', 'vas2nets_sms', 'msgid')
```
Comment (hodgestar) [inline vxvas2nets/vas2nets_sms.py]: Did you intend to use `is_mo_response(...)` here?

**Comment by hodgestar** [comment]
Left one comment, otherwise looks good.

**Comment by justinvdm** [comment]
@hodgestar thanks for review. After testing integration, it doesn't seem like the reply api endpoint is working at the moment, going to change the PR to make this optional.

**Comment by justinvdm** [comment]
Now relies on #4 because of some overlap, otherwise ready for re-review.

**Comment by hodgestar** [comment]
:+1:

File: vxvas2nets/vas2nets_sms.py [hunk 2]
```diff
@@ -23,6 +23,10 @@ class Vas2NetsSmsTransportConfig(HttpRpcTransport.CONFIG_CLASS):
         "null for no timeout",
         default=None)
 
+    reply_outbound_url = ConfigText(
+        "Url to use for reply outbound messages",
+        required=True)
```
Comment (hodgestar) [inline vxvas2nets/vas2nets_sms.py:28]: Just checking that `required=True` is intentional here. I like the idea of asking people to explicitly use `reply_outbound_url: null`, but wasn't sure if that was what was intended.

Comment (justinvdm) [inline vxvas2nets/vas2nets_sms.py:28]: Thanks for catching this, forgot to make this optional, going to change that.

**Pull request merged**